README.md

```diff
@@ -193,7 +193,7 @@ Sample app [example](https://github.com/okta/samples-ios/blob/master/browser-sig
 
 ### signOutOfOkta
 
-You can start the sign out flow by simply calling `signOutFromOkta` with the appropriate `OktaOidcStateManager` . This method will end the user's Okta session in the browser.
+This method will end the user's Okta session in the browser. Method deletes Okta's persistent cookie and disables SSO capabilties 
```

Comment:
This method ends the user's Okta session in the browser. The method deletes Okta's persistent cookie and disables SSO capabilities.